app.json

```diff
@@ -4,7 +4,11 @@
   "cron": [
     {
       "command": "rake top_up_accounts",
-      "schedule": "0 14 * * *"
+      "schedule": "0 14 * * 5"
+    },
+    {
+      "command": "rake check_objectives",
+      "schedule": "0 9-21/2 * * *"
```

Comment:
why only 9-12? why around the clock? I think every hour is ok. or every 30 min.

Reply:
We also could run it on demand in the points of the app that would affect it - on transaction added. And only for the affected account.

Reply:
Good idea, I'll think about it.